analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -221,6 +221,24 @@ def add_arguments_to_parser(parser):
                                     "into the '<OUTPUT_DIR>/success' "
                                     "directory.")
 
+    analyzer_opts.add_argument('--config',
+                               dest='config_file',
+                               required=False,
+                               default=argparse.SUPPRESS,
```

Comment:
do we plan to add a default value for the config file path? `~/.codechecker.json` or something like this what do you think?